### T-test and Chi square test

**Example**

It is assumed that the mean systolic blood pressure is $\mu=120$  mm Hg. In the Honolulu Heart Study, a sample of 
$n = 100$ people had an average systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. Is the group significantly different (with respect to systolic blood pressure!) 
from the regular population?

\begin{equation*}
\begin{cases}
H_0: & \mu=120 \\
H_1: & \mu \neq 120    
\end{cases}
\end{equation*}

If the data is normally distributed then 

\begin{equation*}
T=\dfrac{\bar{X}- \mu}{S/\sqrt{n}} \sim t(n-1)
\end{equation*}

(a t-distribution with $n$ degrees of freedome.)

It makes sense to use the t-statististics 

\begin{equation*}
T=\dfrac{\bar{X}- \mu}{S/\sqrt{n}}
\end{equation*}

We find the value of the *observed test statistics*:






Recall t-student (see last class jupyter notebook) and also
https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.t.html (you can also find the gamma function defintion there.)

In [9]:
import math 
from scipy.stats import t
from scipy.stats import chi2

critical=t.ppf(0.025,99)
critical

-1.9842169515086832

$\alpha=0.05$ significance level

Rejection region: RR=$[-\infty, t_{-\alpha/2}] \cup [t_{\alpha/2}, \infty]=[-\infty, -1.9842] \cup [1.9842, \infty]$


Observed statitics:

In [3]:
t0=(130.1-120)*math.sqrt(100)/21.21
t0

4.761904761904759

$4.7619> t_{-\alpha/2}$ so the observed statistic falls in RR.

P-Value:

In [4]:
pval=2*t.cdf(-t0,99)
pval

6.562701817208617e-06

We reject the null hypothesis $H_0$ since p-value is very small and less than $\alpha=0.05.$

## Chi-square test

Assuming that the data are normally distributed, you use the test for the variance or standard deviation to test whether the population variance or standard deviation is equal to a specified value.


*Test for the variance or standard deviation*


The test statistic 

\begin{equation*}
X^2=\dfrac{(n-1)S^2}{\sigma^2}
\end{equation*}

The test statistics $X^2$ follows a chi-square distribution with $n-1$ degrees of freedom.




About the Chi Square: 

https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.chi2.html

**Example**

A manufacturer of candy must monitor the temperature at which the candies are baked. Too much variation will cause 
inconsistency in the taste of the candy. Past records show that the standard deviation of the temperature has been $1.2^{\circ}$F.

A random sample of 30 batches of candy is selected, and the sample standard deviation of the temperature is $2.1^{\circ}$F

 - a. At the 0.05 level of significance, is there evidence that the population standard deviation has increased above 
 - b. What assumption do you need to make in order to perform this test? 
 - c. Compute the p-value in (a) and interpret its meaning. 
 


\begin{equation*}
\begin{cases}
H_0: & \sigma^2= 1.44  \quad \mbox{ (that is }\sigma=1.2^\circ \, F)\\
H_1: & \sigma^2 > 1.44 \quad \mbox{ (that is }\sigma>1.2^\circ\, F) \\
\end{cases}
\end{equation*}


if you select a level of significance of $\alpha=0.05$, the upper critical values are


In [5]:
chi2score=chi2.ppf(0.95, 29)
chi2score

42.55696780429269

In [6]:
statistic=29*2.1**2/1.44
statistic

88.8125

$X^2$=88.8125;  $X^2> \chi^2_{0.95}$

In [7]:
pvalue=1-chi2.cdf(88.8125,29)
pvalue

5.5389502984226624e-08

The p-value is less than $\alpha=0.05$. We reject $H_0$!!!

Obervation: Recall
Helpful for your final conclusions:

- If the p-value is less than the prescibed $\alpha$, we reject $H_0$.
- If p-value > prescribed $\alpha$, we fail to reject $H_0.$
